pages/linux/scrot.md

```diff
@@ -0,0 +1,15 @@
+# scrot
+
+> Screen capture utility.
+
+- Capture a screenshot:
```

Comment:
Should we add that it dumps the screenshot as a date-stamped file in the current directory?

Reply:
Hrm, maybe. What about this?

```suggestion
- Capture a screenshot and save it to the current directory with the current date as the filename:
```